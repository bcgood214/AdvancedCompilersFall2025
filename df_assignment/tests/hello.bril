@main() {
    v: int = const 2;
    call @print4;
    print v;
}
@print4 {
    v: int = const 4;
    x: int = const 8;
    print v;
    ret;
}