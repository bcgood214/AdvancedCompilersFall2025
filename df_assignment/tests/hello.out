b1:
  in:  âˆ…
  out: ('v', 'b1', 0)
b1:
  in:  âˆ…
  out: ('v', 'b1', 0), ('x', 'b1', 1)
